### DISCUSSION TITLE
v0.13.0-beta.1

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#824] [Releases]
## What's Changed
* [Feature] Public dashboard and Getting Started pages by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/822
* Laravel 10.25.2 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/819
* Release v0.13.0-beta.1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/823

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.12.2...v0.13.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.13.0-beta.1'>v0.13.0-beta.1</a>.</em>

**Comment by jiriteach** [comment]
Awesome work in implementing the public dashboard!
Having formatting issues - looks like it could be missing some assets but nothing showing in the console.

<img width="1413" alt="Screenshot 2023-10-02 at 14 09 50" src="https://github.com/alexjustesen/speedtest-tracker/assets/2116446/b4ff39c3-e929-4461-b471-238a147840f1">

**Reply by alexjustesen** [reply]
Hmmm I bet this is related to the page width config. Probably need to dig into the asset build process. Bet it's excluding some.

**Reply by alexjustesen** [reply]
`v0.13.0-beta.2` in coming which fixes the asset issue https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.13.0-beta.2

**Reply by jiriteach** [reply]
Thanks! - Working well now.

<img width="1532" alt="Screenshot 2023-10-03 at 09 13 32" src="https://github.com/alexjustesen/speedtest-tracker/assets/2116446/02f27df5-c71e-40ee-b545-b5dab7a8182d">

**Reply by alexjustesen** [reply]
Hell yeah!
